sys/cpp11-compat/include/riot/thread.hpp

```diff
@@ -251,6 +255,9 @@ class thread {
 
   ~thread();
 
+  /**
+   * @brief Disallow the assignment operator for constant objects.
+   */
```

Comment:
Same as above and the operator is called [copy assignment operator](http://en.cppreference.com/w/cpp/language/copy_assignment). Move assignment is still allowed using the operator below.